src/services/auth/sign-in/modules/SignInLeftContainer.vue

```diff
@@ -69,7 +69,7 @@ export default {
     height: 100%;
 
     &.admin {
-        @apply bg-primary3;
+        background-color: rgba(210, 210, 246, 0.3);
```

Comment:
```suggestion
        background-color: rgba(theme('colors.xxx'), 0.3);
```
컬러값 변경 시 함께 변경되도록, 컬러토큰 사용이 되어야 합니다.
아마도 피그마에 컬러 토큰이 적혀있을 것으로 예상되는데, 없으면 디쟈너분들께 요청해주시면 됩니다!